src/BitWriter_3_2/BitcodeWriter.cpp

```diff
@@ -0,0 +1,1919 @@
+//===--- Bitcode/Writer/BitcodeWriter.cpp - Bitcode Writer ----------------===//
```

Comment:
Yes, files in src/BitWriter_3_2 and src/BitWriter_3_2.35 are taken verbatim from Android source code tree.
They are responsible for producing bitcode that can be understood by llvm compiler running on the Android device.